dunstctl

```diff
@@ -12,16 +12,19 @@ die(){ printf "%s\n" "${1}" >&2; exit 1; }
 
 show_help() {
 	cat <<-EOH
-	Usage: dunstctl <command> [parameters]"
-	Commands:
-	  close                    Close the last notification
-	  close-all                Close the all notifications
-	  context                  Open context menu of latest notification
-	  history-pop              Pop one notification from history
-	  status                   Check if
-	  status-set [true|false]  Set the status
-	  debug                    Print debugging information
-	  help                     Show this help
+        Usage: dunstctl <command> [parameters]"
+        Commands:
+          action                   Perform the default action, or open the
+                                   context menu of the notification at the
+                                   given position
+          close                    Close the last notification
+          close-all                Close the all notifications
+          context                  Open context menu
+          history-pop              Pop one notification from history
+          status                   Check if
+          status-set [true|false]  Set the status
+          debug                    Print debugging information
+          help                     Show this help
```

Comment:
The indentation-tabs are right now replaced with spaces. To make the `-` of `<<-` useful, these have to be tabs.

Reply:
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07_04